src/components/Common/CrowdsalesList.js

```diff
@@ -29,20 +29,20 @@ export default class CrowdsalesList extends Component {
         <div className="container-fluid">
           <div className="table-row flex-table-header">
             <div className="text">Address</div>
-            <div className="sm-text">Token Name</div>
-            <div className="sm-text">Ticker</div>
-            <div className="sm-text">Date</div>
+            {/*<div className="sm-text">Token Name</div>*/}
+            {/*<div className="sm-text">Ticker</div>*/}
+            {/*<div className="sm-text">Date</div>*/}
           </div>
           <div className="scrollable-content">
-            {crowdsaleStore.crowdsales.map((crowdsale, index) => (
+            {crowdsaleStore.crowdsales.map((crowdsaleAddress, index) => (
               <div className={`table-row ${this.state.selectedRow === index ? 'selected' : ''}`}
                    key={index.toString()}
-                   onClick={() => this.selectCrowdsale(index, crowdsale.contractAddress)}
+                   onClick={() => this.selectCrowdsale(index, crowdsaleAddress)}
               >
-                <div className="text">{crowdsale.contractAddress}</div>
-                <div className="sm-text">{crowdsale.extraData.tokenName}</div>
-                <div className="sm-text">{crowdsale.extraData.ticker}</div>
-                <div className="sm-text">{crowdsale.extraData.creationDate.split('T').join(' ')}</div>
+                <div className="text">{crowdsaleAddress}</div>
+                {/*<div className="sm-text">{crowdsaleAddress.extraData.tokenName}</div>*/}
```

Comment:
@fernandomg can we delete these commented lines before merging to master too?

Reply:
removed